ReflectionAnalyzers/ReflectionAnalyzers.csproj

```diff
@@ -20,18 +24,25 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>0.1.20.0-dev</Version>
+    <Version>0.1.20-dev</Version>
+    <DevelopmentDependency>true</DevelopmentDependency>
     <Description>Analyzers for System.Reflection</Description>
-    <PackageLicenseUrl>https://github.com/DotNetAnalyzers/ReflectionAnalyzers</PackageLicenseUrl>
-    <RepositoryUrl>https://github.com/DotNetAnalyzers/ReflectionAnalyzers</RepositoryUrl>
-    <PackageTags>Reflection Roslyn Diagnostic Analyzer DotnetAnalyzers</PackageTags>
+    <PackageLicenseUrl>http://opensource.org/licenses/MIT</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/DotNetAnalyzers/ReflectionAnalyzers</PackageProjectUrl>
     <PackageReleaseNotes>BUGFIX REFL031: conditional expression.
 BUGFIX REFL031: if statement.</PackageReleaseNotes>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
```

Comment:
❔ If you have this line, do you still need to use `PrivateAssets` on the true dependencies lower in this file?

Reply:
@sharwell Ah, this line is another thing added to minimize changes in behavior that I wondered if we could remove. Without it, a NETStandard.Library dependency is emitted. I don't know if this is good or bad or matters enough to suppress, for an analyzer that doesn't support VS2015 anyway.

PrivateAssets is not redundant because has an effect at a distance on the tests project, for instance. It's also semantically the right thing to do if we decide to stop suppressing the NETStandard.Library dependency.

An alternative to `<SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>` would be `<PackageReference Update="NETStandard.Library" PrivateAssets="true" />`.

Reply:
I think `SuppressDependenciesWhenPacking` is correct when packing analyzers, or tools that do not produce references in the project.

Reply:
> PrivateAssets is not redundant because has an effect at a distance on the tests project, for instance.

It's not redundant, but those dependencies are necessary for testing so it's not a problem to have them.

Reply:
Okay, sounds like a benefit.